packages/nvidia_nat_app/src/nat_app/graph/static_analysis.py

```
@@ -0,0 +1,1147 @@
+                    self._dict_vars[target.id] = keys
+
+            if (isinstance(value, ast.Call) and isinstance(value.func, ast.Name) and value.func.id == "dict"):
+                keys = set()
+                for kw in value.keywords:
+                    if kw.arg is not None:
+                        keys.add(kw.arg)
+                if keys:
+                    self._dict_vars[target.id] = keys
+
+        if isinstance(target, ast.Tuple) and isinstance(value, ast.Tuple):
+            for t, v in zip(target.elts, value.elts):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)